src/reporters/console/console-reporter.js

```diff
@@ -27,9 +27,10 @@ type Row = Array<string>;
 export default class ConsoleReporter extends BaseReporter {
   constructor(opts: Object) {
     super(opts);
-    this._lastCategorySize = 0;
 
+    this._lastCategorySize = 0;
     this.format = (chalk: any);
+    this.isSilent = !!opts.isSilent;
```

Comment:
@bestander Alright, I will give it a shot as soon as I can. Thanks for the feedback 👍